src/libos/src/fs/fs_ops/mount.rs

```diff
@@ -27,5 +28,13 @@ pub fn do_mount_rootfs(
         *root_inode = new_root_inode;
         *ENTRY_POINTS.write().unwrap() = user_config.entry_points.to_owned();
     });
+    let fs_view = FsView::new();
+    let resolv_conf_file = match fs_view.open_file("/etc/resolv.conf", AccessMode::O_RDWR as u32 | CreationFlags::O_CREAT.bits(), 0o666){
+        Err(e) => {
+            return_errno!(EINVAL, "failed to open /etc/resolv.conf in enclave");
+        }
+        Ok(file) => file,
+    };
+    resolv_conf_file.write(&*RESOLV_CONF_BYTES.read().unwrap());
```

Comment:
Should RESOLV_CONF_BYTES be initialized?

Reply:
Yes, since we need to use RESOLV_CONF_BYTES as a mutable global variable in mount.rs,  RESOLV_CONF_BYTES requires lazy initialization.

Reply:
Done